### ISSUE TITLE
Errors while generating index

### ISSUE THREAD

**Issue by Ardakilic** [m2mdas/phpcomplete-extended-laravel#5]
Hello,

Firstly thanks for maintaining these projects, much appreciated!

I have a question, I hope you can help me:

When I try to tun vim and when it starts to index, it throws an error. The error is same with `:PHPCompleteExtendedGenerateIndex` , which is like:

```
Generating autoload classmap
Generating index...
{"error":{"type":"ErrorException","message":"Array to string conversion","file":"\/myLaravelhome\/vendor\/cartalyst\/sentry\/src\/Cartalyst\/Sentr
y\/SentryServiceProvider.php","line":91}}
```

Here's the line that occurs: https://github.com/cartalyst/sentry/blob/master/src/Cartalyst/Sentry/SentryServiceProvider.php#L91

Both this plugin and phpcomplete-extended are installed.

Edit: using Ubuntu 14.04 and PHP 5.5.9 if that would help.

What could be the issue, is there any way to fix this?
Thanks,

**Comment by m2mdas** [comment]
Sorry for very late reply. I am currently out, in fact still am. Tomorrow I will sit in my desk and look into the issue.

**Comment by m2mdas** [comment]
Yes this is a bug, I am able to reproduce the issue. Working on it.

**Comment by m2mdas** [comment]
Please update the `phpcomplete-extended-laravel` and run `:PHPCompleteExtendedGenerateIndex` again.
